### DISCUSSION TITLE
Definitie van Applicatie

### DISCUSSION THREAD

**Discussion by RonaldvCortenberghe** [OpenCatalogi/.github#105] [Q&A]
Hierbij een eerste aanzet (uw kritische bijdrage is gewenst). 

OpenCatalogi is een weergave van Componenten verdeeld over de 5 lagen zoals gedefinieerd door VNG in het Gegevenslandschap (https://www.gemmaonline.nl/index.php/Gegevenslandschap en https://componentencatalogus.commonground.nl/5-lagen-model).

### APPLICATIE
Samenstelling (toepassing) van componenten met minimaal één component op laag 5 ten behoeve van eindgebruik in een gemeentelijk bedrijfsproces/dienstverleningsproces.

In Opencatalogi kan vanuit een applicatie naar de samenstellende componenten worden doorgeklikt, waarbij ook een component kan zijn samengesteld uit meerdere (sub)componenten.

**Comment by RonaldvCortenberghe** [comment]
Bbrands2 wrote:

@RvdKlip Wordt er in de beschrijving met samenstelling van componenten bedoeld dat er minimaal 2 componenten waarvan 1 op laag 5 moet zijn of dat er ook 1 van laag 5 mag zijn?

**Reply by RvdKlip** [reply]
1 Component op laag 5 is voldoende. Met name omdat er waarschijnlijk gemeenten zijn die applicatiesilo’s vanuit het bestaande informatielandschap ook een plek willen geven in OpenCatalogi. Mijn voorstel zou dan zijn om een applicatiesilo in laag 5 zichtbaar te maken.

**Comment by EdwinCoster** [comment]
Is in deze definitie Open Zaak dan (nog) een applicatie? Open Zaak heeft weliswaar een beheerinterface, maar die is niet ten behoeve van eindgebruik in een gemeentelijk bedrijfsproces/dienstverleningsproces.

En is een dienstverleningsproces niet ook een bedrijfsproces?

**Reply by RvdKlip** [reply]
We moeten even afwachten hoe het ontwikkeltraject uitpakt, maar het is de bedoeling dat OpenCatalogi ook een interface heeft voor eindgebruik door bijvoorbeeld informatieadviseurs, portfoliomanagers, architecten en anderen  die geinteresseerd zijn in ‘wat er (te halen) is’.

Inderdaad moeten we dus zorgen dat er geen verwarring (uitsluiting) ontstaat door de typering van deze processen. Zou het helpen als we ‘bedrijfsproces’ veranderen naar ‘bedrijfsvoeringsproces’?

**Reply by EdwinCoster** [reply]
Daarmee is OpenCatalogi een applicatie, maar Open Zaak nog steeds niet. Als je de simpele indeling van het GEMMA gegevenslandschap pakt en onderscheidt maakt tussen dienstenafnemers, integratiefunctionaliteit en dienstenaanbieders, dan zijn combinaties van componenten   op het niveau van dienstenaanbieders geen applicaties. Dat is prima, maar wat zijn het dan wel?

En op het procesgebied krijgen we dan bedrijfsvoeringsproces/dienstverleningsproces.
